layouts/partials/team.html

```diff
@@ -63,7 +63,7 @@ <h5 class="modal-title" id="{{ $id }}Title">{{ .name }}</h5>
                     </div>
                     <div class="modal-body">
                       {{ .description | markdownify }}
-                      <img src="{{ $image300.RelPermalink }}" class="img-fluid">
+                      <img src="{{ $image300.RelPermalink }}" class="img-fluid mt-3">
```

Comment:
Does the `mt-3` part do anything special? 

(If it makes me look better then LGTM 😛 )

Reply:
Adds some spacing between the text and the image (margin-top) this is too little:
![image](https://user-images.githubusercontent.com/1254351/203578943-787a350a-8f10-49a0-b5af-cbe08212a131.png)

Reply:
Ah I see. And I agree the extra spacing looks better. Can you also add it on line 133 (Alumni section) of the same file then? 😄

Reply:
I can also add it in in the rework I'm doing of the alumni section. I'll just merge this. 👍